packages/subgraph/src/entities/Global.ts

```diff
@@ -96,6 +96,14 @@ export function increaseNumberOfLiquidatedTroves(): void {
   global.save();
 }
 
+export function decreaseNumberOfLiquidatedTroves(): void {
+  let global = getGlobal();
+
+  global.numberOfLiquidatedTroves--;
+  global.numberOfOpenTroves++;
```

Comment:
Why does the number of Troves increase when a Trove is liquidated?

Reply:
I see - this function is called when a Trove is "re-opened" where it was previously in liquidated state